backend/src/main/java/com/soen341/instagram/security/UserDetailsServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.soen341.instagram.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import com.soen341.instagram.dao.impl.AccountRepository;
+
+@Service("UserDetailsService")
+public class UserDetailsServiceImpl implements UserDetailsService
```

Comment:
If my previous comment is to be implemented, we'd need to change this to `UserDetailsServiceImplementation`